client/components/details-link/index.tsx

```diff
@@ -0,0 +1,53 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import InfoOutlineIcon from 'gridicons/dist/info-outline';
+import { Link } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { getAdminUrl } from 'wcpay/utils';
+
+/**
+ * The parent segment is the first part of the URL after the /payments/ path.
+ */
+type ParentSegment = 'deposits' | 'transactions' | 'disputes';
```

Comment:
Just pointing out that `transactions` is only used in a unit test but no real use of `DetailsLink` that passes `transactions` to the `parentSegment` attribute.